### PR TITLE
Added new examples for manually testing AmpContext

### PR DESCRIPTION
**Pull request by bradfrizzell**
Added new example pages for more comprehensive manual testing of the AmpContext library.

loads-ampcontext-creative-runtime-not-at-top.html loads the AMP runtime in an iframe so that it is not at top, and then loads a dummy creative that uses AmpContext. This is to make sure that messaging works properly when the runtime is not at top.

creates-nested-ampcontext-creative.html is used inside of loads-windowcontext-creative.html to make a deeply-nested creative, to make sure that messaging is working properly from within a deeply nested ad.

### CODE REVIEW

File: examples/load-ampcontext-creative-runtime-not-at-top.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<html>
```
Comment (lannka) [inline examples/load-ampcontext-creative-runtime-not-at-top.html]: As i see such a requirement more and more often (to test a page inside an iframe).
I suggest we introduce a new endpoint in our `server.js`.

it should handle request like `/iframe/examples/loads-windowcontext-creative.max.html`, and dynamically return this HTML.

Let me know if you don't know how to do it, I can help.

File: examples/loads-windowcontext-creative.html [hunk 2]
```diff
@@ -25,6 +25,11 @@ <h1 style="font-size: 1em;"> window.context example </h1>
             src="http://localhost:8000/examples/ampcontext-creative.html">
     </amp-ad>
     <div id="filler_space" ></div>
+    <amp-ad width="300" height="400"
+            type="fake"
+            data-use-a4a="true"
+            src="http://localhost:8000/examples/creates-nested-ampcontext-creative.html">
```
Comment (lannka) [inline examples/loads-windowcontext-creative.html]: can we avoid hardcoded host names?

Comment (bradfrizzell) [inline examples/loads-windowcontext-creative.html]: done

File: examples/load-ampcontext-creative-runtime-not-at-top.html (hunk 1 above)
Comment (bradfrizzell) [inline examples/load-ampcontext-creative-runtime-not-at-top.html]: Done https://github.com/ampproject/amphtml/pull/7615

File: examples/creates-nested-ampcontext-creative.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<!doctype html>
```
Comment (lannka) [inline examples/creates-nested-ampcontext-creative.html]: what do you think if we introduce another endpoint for this:
`/nameframe/ampcontext-creative.html`

so we can have nested nameframes like this:
`/nameframe/nameframe/ampcontext-creative.html`

Comment (bradfrizzell) [inline examples/creates-nested-ampcontext-creative.html]: What would the use case be for that?

**Comment by bradfrizzell** [comment]
Is there anything missing on this PR?

**Pull request merged**